src/web/screens/sharingPermissions.tsx

```diff
@@ -105,6 +103,19 @@ function SharingPermissions() {
     loadSharingList();
   }, [loadSharingList]);
 
+  const renderSharingPermissionTable = () => {
+    if (!participant?.completedRecommendations) return;
+    const tableSharedSiteIds = sharedSiteIds ?? [];
+    const tableSharedTypes = sharedSiteIds === null ? ['DSP' as ClientType] : sharedTypes;
```

Comment:
This breaks type safety - you can put anything in that string. 
![image](https://github.com/IABTechLab/uid2-self-serve-portal/assets/111951647/defdc386-f594-4b46-a09c-91fedeb3dae1)
Please don't use `as` unless you really have to.

Reply:
Thanks! I removed the type assertion and add type annotation to tableSharedTypes